azurelinuxagent/common/event.py

```diff
@@ -52,6 +52,7 @@ class WALAEventOperation:
     Disable = "Disable"
     Download = "Download"
     Enable = "Enable"
+    ExtensionProcessing = "ExtensionHandling"
```

Comment:
Why is the constant name different from the value?

Reply:
I can change that.